doc/source/user/absolute_beginners.rst

```diff
@@ -1339,52 +1382,54 @@ remember to include a docstring with your function using a string literal
 For example, if you create this function::
 
   >>> def double(a):
-  >>>   '''Return a * 2'''
-  >>>   return a * 2
+  ...   '''Return a * 2'''
+  ...   return a * 2
 
-You can run::
+You can obtain information about the function:
 
-  >>> double?
+.. code-block:: ipython
 
-Which will return::
-
-  Signature: double(a)
-  Docstring: Return a * 2
-  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
-  Type:      function
+  In [2]: double?
+  Out[2]: Signature: double(a)
+          Docstring: Return a * 2
+          File:      ~/Desktop/<ipython-input-23-b5adf20be596>
+          Type:      function
 
 You can reach another level of information by reading the source code of the
 object you're interested in. Using a double question mark (``??``) allows you to
 access the source code.
 
-For example, running::
-
-  >>> double??
+For example:
 
-Will return ::
+.. code-block:: ipython
 
-  Signature: double(a)
-  Source:    def double(a):
-                '''Return a * 2'''
-                return a * 2
-  File:      ~/Desktop/<ipython-input-23-b5adf20be596>
-  Type:      function
+  In [3]: double??
+  Out[3]: Signature: double(a)
+          Source:    def double(a):
```

Comment:
When I do this, the source code is follows the title `Source:` and is aligned on the left margin:

```
Signature: double(a)
Source:   
def double(a):
    '''Return a*2'''
    return a*2
File:      ~/<ipython-input-4-c55279f3f8ac>
Type:      function
```

Reply:
This is what I was getting at